premium/api-keys.md

```diff
@@ -0,0 +1,169 @@
+3. Your new API Key and Secret will be generated and displayed
+4. Update credentials on all your devices
+
+::: tip When to Regenerate
+
+- If you suspect your credentials have been compromised
+- If you want to revoke access from all devices
+  :::
+
+## Security Best Practices
+
+### Protecting Your Credentials
+
+::: danger Keep Your Secret Safe
+
+- **Never** share your API Secret with anyone
+- **Never** post it in public forums or Discord
+- **Never** commit it to version control (Git, etc.)
+- **Never** send it via unencrypted channels
+  :::
+
+### Storage Recommendations
+
+Store your API credentials securely:
+
+- Use a password manager (1Password, Bitwarden, KeePass, etc.)
+- Keep backups in a secure location
+- Don't save them in plain text files
+- Don't email them to yourself
+
+### If Credentials are Compromised
+
+If you believe your API credentials have been exposed:
+
+1. **Immediately** regenerate your API credentials from the [subscription page](https://rotki.com/home/subscription) by clicking **Regenerate**
+2. Update all your legitimate devices with the new credentials
+3. Monitor your account for unusual activity
+
+## API Keys and Device Limits
+
+Each premium tier has a device limit. When you add your API credentials to rotki:
+
+- The device is registered to your account
+- It counts toward your device limit
+- You can view and manage devices from the [devices page](/premium/devices)
+
+### Checking Device Usage
+
+From your [subscription page](https://rotki.com/home/subscription):
+
+- View current device count
+- See your tier's device limit
+- Access the [device list](https://rotki.com/home/devices) for details
+
+### Device Limit Exceeded
+
+If you reach your device limit:
+
+- New devices cannot authenticate
+- You'll see an error in the rotki app
+- Remove unused devices to free up slots
+
+See [Device Management](/premium/devices) for details on managing devices.
+
+## Troubleshooting
+
+### Invalid API Key/Secret Error
+
+If you see authentication errors:
+
+1. Verify you copied the credentials correctly (no extra spaces)
+2. Ensure your subscription is active (check [subscription page](https://rotki.com/home/subscription))
+3. Verify your API credentials are still visible on the subscription page
+4. Try regenerating your credentials
+
+### Can't See My API Key/Secret
+
+If you can't see your credentials on the subscription page:
+
+1. Verify you're logged in to the correct rotki.com account
+2. Ensure you've already generated API credentials (if not, click the generate button)
+3. Refresh the page
+4. Contact [info@rotki.com](mailto:info@rotki.com) if the issue persists
+
+### Premium Features Not Working
+
+If premium features aren't activating:
+
+1. Verify API credentials are entered correctly in the app
+2. Check your subscription is active and not expired
+3. Ensure you haven't exceeded your device limit
+4. Try removing and re-adding the credentials
+5. Check the app logs for specific errors
```

Comment:
i would share an screenshot here about how it looks like when it correctly works